README.md

```diff
@@ -16,43 +16,86 @@ Our goal is to provide a substrate which can be built upon and extended to serve
 debugging and maintaining their serverless application.
 
-## Documentation
+## Dispatch Solo
 
-Checkout the detailed [documentation](https://vmware.github.io/dispatch) including a [quickstart guide](https://vmware.github.io/dispatch/documentation/guides/quickstart).
+Dispatch Solo is a simplified branch of Dispatch.  The objective is to remove as many dependencies as possible
+while maintaining functionlity.  Dispatch Solo is a single binary which only relies on Docker as dependency.
+The trade-off of this simplification is scale and persistence.  However, it is the perfect platform for evaluation
+and learning.
+
+## Documentation and Quickstart
+
+Checkout the detailed [documentation](https://vmware.github.io/dispatch) including a [quickstart guide](https://vmware.github.io/dispatch/documentation/front/quickstart).
 
 ## Architecture
 
 The diagram below illustrates the different components which make up the Dispatch project:
 
-![initial dispatch architecture diagram](docs/_specs/dispatch-v1-architecture.png "Initial Architecture")
+![dispatch solo architecture diagram](docs/assets/images/dispatch-solo-architecture.png "Dispatch Solo Architecture")
 
 ## Installation
 
-Installing Dispatch is easy once you have a compatible Kubernetes installation.  For instance, to deploy on minikube:
+For a quickstart see the [developer documentation](#documentation-and-quickstart)
+
+## Building
+
+### Prerequisites
 
-1. Fetch the IP address of minikube as this will be used the host for dispatch services.
+* Golang
+* Docker
+
+Clone the repository and update your GOPATH:
+
+```bash
+mkdir -p ~/dispatch/go/src/github.com/vmware
+git clone git@github.com:vmware/dispatch.git
+cd dispatch
+git checkout origin/solo
+export GOPATH=~/dispatch/go
 ```
-export DISPATCH_HOST=$(minikube ip)
+
+Build Dispatch CLI and Server (Mac):
+
+```bash
+make darwin
+cp ./bin/dispatch-darwin /usr/local/bin/dispatch
+./bin/dispatch-server-darwin
 ```
 
-2. Configure the installation:
+Build Dispatch CLI and Server (Linux):
+
+```bash
+make linux
+cp ./bin/dispatch-linux /usr/local/bin/dispatch
+./bin/dispatch-server-linux
 ```
-$ cat << EOF > config.yaml
-apiGateway:
-  host: $DISPATCH_HOST
-dispatch:
-  host: $DISPATCH_HOST
-  debug: true
-  skipAuth: true
+
+Create dispatch config file $HOME/.dispatch/config.json like this:
+```bash
+mkdir ~/.dispatch
+cat << EOF > $HOME/.dispatch/config.json
+{
+    "current": "solo",
+    "contexts": {
+        "solo": {
+            "port": 8080,
+            "scheme": "http",
+            "organization": "dispatch-server",
+            "cookie": "cookie",
+            "insecure": true
+        }
+    }
+}
 EOF
 ```
 
-3. Install Dispatch:
-```
-$ dispatch install --file config.yaml
-```
+## Testing
+
+To run the end to end tests:
 
-For a more complete quickstart see the [developer documentation](#documentation)
+```bash
+API_GATEWAY_HTTP_HOST=http://localhost:8081 ./e2e/scripts/run-e2e.sh e2e/tests
```

Comment:
got `You must specify a dispatch config file if installing dispatch (default).` when run this cmd